.github/workflows/publish.yml

```
@@ -9,7 +9,38 @@ permissions:
   contents: write
 
 jobs:
+  version:
+    name: Update version
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.9
+      - name: Install poetry
+        run: |
+          pip install --upgrade poetry
+      - name: Update version
+        run: |
+          VERSION=${GITHUB_REF#refs/tags/v}
+          poetry version $VERSION
+      - name: Commit changes
+        run: |
+          git config --local user.email "${{ secrets.GIT_USER_EMAIL }}"
+          git config --local user.name "${{ secrets.GIT_USER_NAME }}"
+          git commit -am "Update version"
+          
+      - name: Push changes
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch: ${{ github.ref }}
```

```suggestion
          git commit -am "chore: Update version"
          git push
```

this way we're ready for release please
and we reduce our dependencies on third party